flow/include/flow/Arena.h

```diff
@@ -853,6 +853,40 @@ inline bool operator>=(const StringRef& lhs, const StringRef& rhs) {
 	return !(lhs < rhs);
 }
 
+typedef uint64_t Word;
+// Get the number of prefix bytes that are the same between a and b, up to their common length of cl
+static inline int commonPrefixLength(uint8_t const* ap, uint8_t const* bp, int cl) {
```

Comment:
This seems like a function move from DeltaTree -> Arena; does it have any associated unit test?

Reply:
Not directly, at least not anymore, but it's indirectly tested by most of the redwood unit tests.

Reply:
Ahh, makes sense. 
Given it is a Arena util method, having a dedicated test would help in a longer term, but, it can be done as a separate PR.

Reply:
Please add a TODO.